test/csuite/schema_abort/main.c

```diff
@@ -231,38 +231,24 @@ test_bulk(THREAD_DATA *td)
     WT_CURSOR *c;
     WT_DECL_RET;
     WT_SESSION *session;
-    bool create;
 
     testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
 
     if (use_txn)
-        testutil_check(session->begin_transaction(session, NULL));
-    create = false;
+        testutil_die(ret, "test_bulk should not be used with transactions");
```

Comment:
Instead of crashing here and adding another conditional elsewhere, why not just return here and have it be a no-op? In the current change, if `use_txn` is set it is a no-op at the higher level. That keeps the `thread_run` code cleaner and easier to read/understand and puts the logic into the operations.

Reply:
I see your point and that's also what @keitharnoldsmith suggested on the ticket. I decided to go a different way because if someone reads what the thread does at the higher-level, it can be misleading. But I am ok with what you both are suggesting as well! See a26eb92fe7e02d9be96f0ae56d14a0e4c550e03c.

Reply:
My first changes were wrong, please see 004abb5b1cb9ae6213f7c452eb1aa49a448ce34c as well.

Reply:
Thanks @etienneptl. Another reason to have this logic in the `test_bulk` functions is for consistency. 

Some operations aren't supported on tiered storage. And the logic for not testing those operations on tiered storage is in the corresponding test functions, such as `test_drop` and `test_verify`.